### PR TITLE
added ID info to fn list output in app, funcs & triggers

### PR DESCRIPTION
**Pull request by nishalad95**
fix: https://github.com/fnproject/cli/issues/449

### CODE REVIEW

**Comment by zootalures** [comment]
Looks like we have some (perhaps over-sensitive) tests that don't like the new format.

File: testharness/harness.go [hunk 1]
```diff
@@ -403,13 +403,15 @@ runtime: docker
 //NewFuncName creates a valid function name and registers it for deletion
 func (h *CLIHarness) NewFuncName(appName string) string {
 	funcName := randString(8)
+	//funcID := randString(26)
```
Comment (zootalures) [inline testharness/harness.go]: Can you delete these

Otherwise gtg

File: testharness/harness.go [hunk 2]
```diff
@@ -403,13 +403,15 @@ runtime: docker
...
 	h.funcRefs = append(h.funcRefs, funcRef{appName, funcName})
 	return funcName
 }
 
 //NewTriggerName creates a valid trigger name and registers it for deletioneanup
 func (h *CLIHarness) NewTriggerName(appName, funcName string) string {
 	triggerName := randString(8)
+	//triggerID := randString(26)
```
Comment (zootalures) [inline testharness/harness.go]: Delete this

**Comment by nishalad95** [comment]
Sure, ...I was in the process of making a test to check the IDs, but if it's not needed I'll make the changes and commit

**Pull request merged**